test/fhirdefs/load.test.ts

```diff
@@ -46,10 +46,11 @@ describe('#loadCustomResources', () => {
     expect(profiles[2].id).toBe('MyObservation');
     expect(valueSets).toHaveLength(1);
     expect(valueSets[0].id).toBe('MyVS');
-    // Each extension has 3 entries, one for url, one for id, and one for name
-    expect(extensions).toHaveLength(6);
-    const birthPlace = extensions.find(e => e.id === 'patient-birthPlace');
-    const birthPlaceFromXML = extensions.find(e => e.id === 'patient-birthPlaceXML');
+    expect(extensions).toHaveLength(2);
```

Comment:
Oh this is a nice feature from FPL!

Reply:
Yes, you can thank @mint-thompson for that one! In fact, it looks like [you already did](https://github.com/standardhealth/fhir-package-loader/pull/6#pullrequestreview-992448535)!